11LessonEleven/in-class-project/index.html

```diff
@@ -15,28 +15,28 @@
     <header class="header">
         <h1>Welcome to our site</h1>
     </header>
-    <main class="clearfix">
+    <main class="clearfix main">
 
         <div class="box">
             <div class="blue"></div>
             <div class="container">
                 <h1>Title</h1>
-                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. Biscuit danish jelly-o jelly-o cupcake candy powder chocolate pie. </p>
+                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. </p>
             </div>
         </div>
 
         <div class="box">
             <div class="orange"></div>
             <div class="container">
                 <h1>Title</h1>
-                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. Biscuit danish jelly-o jelly-o cupcake candy powder chocolate pie. </p>
+                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. </p>
           </div>
         </div>
         <div class="box">
             <div class="aqua"></div>
             <div class="container">
                 <h1>Title</h1>
-                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. Biscuit danish jelly-o jelly-o cupcake candy powder chocolate pie. </p>
+                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. </p>
             </div>
         </div>
     </main>
```

Comment:
Just need some CSS on that H1 in the phone media query. Let's get that fixed and I can bump your grade up.
![screen shot 2016-09-13 at 6 05 24 pm](https://cloud.githubusercontent.com/assets/12698279/18494652/1440e3a6-79dd-11e6-984f-0efdd0a31453.png)